include/functions.h

```diff
@@ -245,6 +248,7 @@ s32 npc_test_move_simple_with_slipping(s32, f32*, f32*, f32*, f32, f32, f32, f32
 s32 npc_test_move_complex_with_slipping(s32, f32*, f32*, f32*, f32, f32, f32, f32);
 
 Npc* get_npc_by_index(s32 listIndex);
+void free_npc_by_index(s32 listIndex);
```

Comment:
This is already declared in `npc.h` - include that whereever you're calling `free_npc_by_index` instead.